docs/blog/pydantic-2-alpha.md

```diff
@@ -0,0 +1,40 @@
+# Pydantic 2 Alpha Release
+
+<aside class="blog" markdown>
+![Samuel Colvin](/img/samuelcolvin.jpg)
+<div markdown>
+  **Samuel Colvin** &bull;&nbsp;
+  [:material-github:](https://github.com/samuelcolvin) &bull;&nbsp;
+  [:material-twitter:](https://twitter.com/samuel_colvin) &bull;&nbsp;
+  :octicons-calendar-24: April 01, 2023 &bull;&nbsp;
+  :octicons-clock-24: 5 min read
+</div>
+</aside>
+
+---
+
+We're excited to announce the first alpha release of Pydantic 2!
+
+This first Pydantic 2 alpha no April Fool's joke. After months of work, we invite you to explore the improvements we've made and give us your feedback. We look forward to hearing your thoughts and working together to improve the library.
+
+For many of you, Pydantic is already a key part of your Python toolkit and needs no introduction &mdash; we hope you'll find the improvements and additions in Pydantic 2 useful.
+
+If you're new to Pydantic: Pydantic is an open-source Python library that provides powerful data parsing and validation &mdash; including type coercion and useful error messages when typing issues arise &mdash; and settings management capabilities. See [the docs](/) for examples of Pydantic at work
+
+Note that there are still some rough edges and incomplete features, and while using the Pydantic 2 alpha releases you may experience errors. If you do encounter any issues, please [create an issue in GitHub](https://github.com/pydantic/pydantic/issues) using the `bug V2` label. This will help us to actively monitor and track errors, and to continue to improve the library’s performance.
```

Comment:
```suggestion
Note that there are still some rough edges and incomplete features, and while trying out the Pydantic 2 alpha releases you may experience errors. If you do encounter any issues, please [create an issue in GitHub](https://github.com/pydantic/pydantic/issues) using the `bug V2` label. This will help us to actively monitor and track errors, and to continue to improve the library’s performance.
```
I sort of feel like this helps set expectations a bit better :)